Makefile

```diff
@@ -45,16 +48,70 @@ server-shell: compose-build
 	@docker-compose run $(IMAGE_NAME) /bin/bash
 
 integration-shell: compose-build
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml run $(IMAGE_NAME) /bin/bash
+ifeq (,$(env))
+	@echo "no environment specified"
+	@path=""; \
+	for word in $(DOCKERFILE_ENVIRONMENTS); do \
+		path="$$path -f docker-compose.integration-$$word.yml"; \
+	done; \
+	docker-compose -f docker-compose.yml $$path build; \
+	docker-compose -f docker-compose.yml $$path up -d; \
+	sleep 5; \
```

Comment:
This seems to run fine with a 5 second sleep but we might need to bump it up

Reply:
These kinds of sleeps are unfortunate. I used `pg_isready` in fidesdemo to avoid it (https://github.com/ethyca/fidesdemo/blob/main/Makefile#L88), but that only works for Postgres.

I don't have any other ideas, but for clarity I'd recommend at least putting an echo statement like "Pausing for 5s to allow all integration databases to be ready..."